src/Main.java

```diff
@@ -1,5 +1,62 @@
+import org.w3c.dom.ls.LSOutput;
+
+import java.time.LocalDate;
+
 public class Main {
+    public static boolean determineYear(int year) {
+        boolean yearLeap;
+        yearLeap = (((year % 4 == 0) && (year % 100 != 0)) || (year % 400 == 0));
+        return yearLeap;
```

Comment:
```suggestion
        return year % 4 == 0 && year % 100 != 0 || year % 400 == 0;
```
лучше всегда стараться писать код короче. Также много лишних скобок